test/parallel/test-v8-serdes.js

```diff
@@ -74,6 +75,15 @@ const deserializerTypeError =
     ser.writeUint32(buf.length);
     ser.writeRawBytes(buf);
 
+    // `v8.serializer.writeRawBytes()` should support both `TypedArray`
+    // and `DataView`.
+    const arrayBufferViews = common.getArrayBufferViews(buf);
+    arrayBufferViewsLength = arrayBufferViews.length;
```

Comment:
Could this be `const arrayBufferViewsLength` instead.
.
.
.
Which made be think isn't there a race here?
Is the callbacks guaranteed to be in order?

Reply:
> Is the callbacks guaranteed to be in order?
Yes. We have only one `HostObject` so that the callback will be called only once. 

I think `v8.Serializer.writeRawBytes()` being expected to be called inside of the callback of `_writeHostObject` makes the test a bit confusing. I'm going to refactor this.